src/assets/sass/protocol/includes/_mixins.scss

```diff
@@ -234,9 +234,14 @@
         text-decoration: underline;
     }
 
+    a:visited {
+        opacity: 0.8;
```

Comment:
I haven't tested, but might this turn buttons translucent? If not then this is great, but if it does maybe we just need an explicit `opacity: 1;` override in the button atom. Or alternatively we could try `color: transparentize($color-white, .2);`?

Reply:
You're so smart. 🤯